docs/releases/4.2.9.txt

```diff
@@ -11,3 +11,7 @@ Bugfixes
 
 * Fixed a regression in Django 4.2.8 where admin fields on the same line could
   overflow the page and become non-interactive (:ticket:`35012`).
+
+* Fixed a regression in Django 4.2 that caused a crash of
+  ``QuerySet.count()`` when called on a queryset combined through methods such
+  as ``QuerySet.union()`` (:ticket:`35042`).
```

Comment:
Django 4.2 is in extended supported so it no longer receives bugfixes (except security patches). We backported patch for #35012 only because it was a regression in the last mainstream support release.

Reply:
Makes sense, do we plant to backport to 5.X or should I drop the release note entirely?

Reply:
Entirely :upside_down_face:

Reply:
Done.